src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.cs

```diff
@@ -40,5 +40,19 @@ private static void TryLoadManagedAssembly()
                 // ignore
             }
         }
+
+        private static string ReadEnvironmentVariable(string key)
+        {
+            try
+            {
+                return Environment.GetEnvironmentVariable(key);
+            }
+            catch
+            {
+                // Unfortunately we don't have a way to log those yet
```

Comment:
What's your opinion on logging errors to stdout or stderr in cases like these? Although it's probably useless in IIS applications, I think it's common practice for console apps, especially on Linux.

Reply:
A possiblility:
https://github.com/DataDog/dd-trace-dotnet/pull/748

Reply:
Now using the StartupLogger, thanks for that!